src/Microsoft.DotNet.Wpf/src/PresentationCore/OtherAssemblyAttrs.cs

```diff
@@ -18,7 +18,7 @@
 [assembly:InternalsVisibleTo(BuildInfo.PresentationFrameworkSystemDrawing)]
 [assembly:InternalsVisibleTo(BuildInfo.SystemWindowsControlsRibbon)]
 [assembly:InternalsVisibleTo(BuildInfo.WindowsFormsIntegration)]
-
+[assembly:InternalsVisibleTo("PresentationCore.Tests,PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
```

Comment:
Is this the normal way of making tests work? Is this public key value stable for all people who clone, build, and test?

Reply:
@jonwis Yes, the public key value is stable for all other's build.
It is used in a similar manner within the repository.
https://github.com/dotnet/wpf/blob/44e26b44141f81af2004330681de835a36e5d07b/src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/OtherAssemblyAttrs.cpp